couchdb/10-docker-default.ini

```diff
@@ -18,6 +18,15 @@ bind_address = 0.0.0.0
 server_options = [{recbuf, 262144}]
 socket_options = [{sndbuf, 262144}, {nodelay, true}]
 require_valid_user = true
+authentication_handlers = {chttpd_auth, cookie_authentication_handler}, {chttpd_auth, proxy_authentication_handler}, {chttpd_auth, default_authentication_handler}
+
+[chttpd_auth]
+require_valid_user = true
+secret = 5ec56423-0856-4b13-8ab7-9b476fea2c39 ; CHANGE ME  - proxy secret
+x_auth_roles = X-Auth-CouchDB-Roles ; http header name for passing the comma separated couch user roles
```

Comment:
It looks like you just kept the default values for these header names. As far as I know, the default values should work fine for our case, but do we even need to include this config here if we just use the default values?